lumicks/pylake/fitting/models.py

```diff
@@ -259,7 +259,7 @@ def odijk(name):
     )
 
 
-def dsdna(name, kilobase_pairs, kilobase_pairs_per_micron=0.34, temperature=24.53608821):
+def dsDNA(name, dna_length_kbp, um_per_bp=0.34, temperature=24.53608821):
```

Comment:
I think it should be in line with python conventions and be named `dsdna`.

Reply:
I suspect it's either `um_per_kbp` or `nm_per_bp` , but not `um per bp` (I prefer the former since we work with microns for Lc).

Reply:
I'll see if I can understand this first indeed. These were based on the citation. :ok_hand:

Reply:
Fixes look good.